lightning_pose/models/heatmap_tracker.py

```diff
@@ -209,7 +218,7 @@ def forward(
     def get_loss_inputs_labeled(self, batch_dict: HeatmapBatchDict) -> dict:
         """Return predicted heatmaps and their softmaxes (estimated keypoints)."""
         # images -> heatmaps
-        predicted_heatmaps = self.forward(batch_dict["images"])
+        predicted_heatmaps = self.forward(batch_dict["images"], do_context=self.do_context)
```

Comment:
if `do_context` is an attribute of `self` it doesn't need to go as argument to `self.forward()`. inside the latter, just do `if self.do_context: ...`
Maybe use the name `self.do_temporal_context`?

Reply:
This is a bit tricky, but currently, in the get_loss_inputs_unlabeled, I had to set `do_context=False ` since the dataloader doesn't actually have context. So I made it a variable of the forward.